app/Database/Migrations/2024-10-10-192752_AddTalk.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Database\Migrations;
+
+use CodeIgniter\Database\Migration;
+
+class AddTalk extends Migration
+{
+    public function up()
+    {
+        // A talk is connected to an event and a speaker (both are foreign keys). It need's
+        // data about the starting date and time, the duration, the title, and a description.
+        $this->forge->addField([
+            'id' => [
+                'type' => 'INT',
+                'unsigned' => true,
+                'auto_increment' => true,
+            ],
+            'event_id' => [
+                'type' => 'INT',
+                'unsigned' => true,
+                'null' => false,
+            ],
+            'speaker_id' => [
```

Comment:
Im not quite sure how the Talkrunden will work.
Will there be a separate model for this?
Or what is your plan?

Reply:
I have not thought about this. But I could think of the following solution:
I create a separate table that allows us to *add* people to an existing talk. If (and only if) a talk has additional speakers, I will only pass those to you, but not the "original" speaker of the talk.
That means: The admins (either you or me) can create a talk where we assign any speaker (I think ourselves, basicly). And then we will add additional people via the new table.
Of course, we will then have to change the data structure so that it's not `speaker_id` anymore, but rather `speaker_ids` (which then would be an array).

Reply:
do that.

Reply:
This will be tracked in #39.